src/gg.regression.unity.bots/Editor/Scripts/BotManagement/IRGBotFinder.cs

```diff
@@ -0,0 +1,71 @@
+using UnityEditor;
```

Comment:
I notice this file is in the Editor folder, but wouldn't we also want this for builds that QA developers are using on a device?